packages/cx-navigation/src/components/BurgerMenu/tokens/cxBurgerMenu.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { asStatic } from '@bodiless/core';
+import { cxColor } from '@bodiless/cx-elements';
+import {
+  addClassesIf,
+  as,
+  flowHoc,
+} from '@bodiless/fclasses';
+import { useIsBurgerMenuHidden } from '@bodiless/navigation';
+import { cxMenu } from '../../Menu';
+import { cxMenuToggler } from '../../MenuToggler';
+import { asBurgerMenuToken } from '../BurgerMenuClean';
+
+/**
+ * Token that provides Push animation to the Burger Menu.
+ */
+const WithBurgerMenuPush = asBurgerMenuToken({
```

Comment:
Why do we have this in a separate token?  I would just move it to Behavior domain of main token.  But if we leave it, should probably rename it since it no longer actually pushes anthything, right?